packages/adconnectdump.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'ADConnectDump'
+$category = 'Cloud'
```

Comment:
As in https://github.com/mandiant/VM-Packages/pull/253, this is another category we don't have at the moment (see https://github.com/mandiant/VM-Packages/issues/38). We need a linter and further discussion on this.